examples/kitty/client/kitty.py

```diff
@@ -325,23 +361,28 @@ async def main():
 
 
 def help_prompt():
-    print("\nWelcome to Kitty!\nUsage: kitty.run(String IP_ADDRESS).\n\
-            IP_ADDRESS: The IP address of the kitty server.")
+    print("\nWelcome to Kitty!\nUsage: kitty.run(String HOST).\n\
+            HOST: The host name / IP address of the kitty server.")
 
 
-def run(ip="172.16.0.2"):
+def run(host="fridge.keg.cse.unsw.edu.au"):
```

Comment:
And not too sure if we want to merge our host name into main, although that is what we've done in the past.